html-forms/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get" class="form-example">
+      <div>
+        <label for="email">Email: </label>
+        <input type="text" name="name" id="name" required />
+      </div>
+    </form>
+    <form action="http://www.example.com/profile.php">
+      <div>
+        <label for="Why">Why is coding cool?</label>
+        <div>
+          <select name="devices">
+            <option value="Select">Select an option</option>
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
```

Comment:
The `for` attribute of the `label` needs to match an `id` of the `select`. So you should add an `id` to this `select` in order to match the `for` attribute